tests/node_converters/binary_operations_test.py

```diff
@@ -8,7 +8,7 @@
 
 @pytest.mark.parametrize(
     'op_type',
-    ('Add', 'Sub', 'Mul', 'Div'),
+    ('Add', 'Sub', 'Mul', 'Div', 'Mod'),
```

Comment:
Because you created a separate test `tests/node_converters/mod_test.py`, you don't need to modify this test.